extensions/amp-form/0.1/amp-form.js

```diff
@@ -1127,6 +1127,11 @@ export class AmpForm {
    * @private
    */
   maybeHandleRedirect_(response) {
+    const doc = this.form_.ownerDocument;
+    if (doc && isAmp4Email(doc)) {
+      user().warn('Redirects not supported in AMP4Email.');
```

Comment:
I think this should be an error. Also I believe this API requires the tag as the first parameter.